code/formatters/spdx.py

```diff
@@ -89,19 +96,29 @@ def format_report(cls, report: DocumentReport):
             finding_annotation.annotation_type = "OTHER"
             finding_annotation.comment = finding.json()
             SPDX_document.add_annotation(finding_annotation)
+
         dependent_documents = []
         for dependency in report.dependencies:
             dep_document = create_base_spdx_document(dependency.name)
-            dep_document.package = Package(name=dependency.name, spdx_id=dep_document.spdx_id, download_location=dependency.download_location, version=dependency.version)
+            dep_document.package = Package(name=dependency.name, spdx_id=dep_document.spdx_id,
+                                            download_location=dependency.download_location, version=dependency.version)
+            checksums = create_spdx_namespace(dependency.name)
```

Comment:
Again I am unclear why this is being used for hashing, as it was already an existing function with a specific purpose. Please revert this change.